src/clj/rems/db/applications.clj

```diff
@@ -924,6 +935,9 @@
        :event/time time
        :event/actor actor
        :application/id application-id
+       :application/external-id (when allocate-external-id? ;; TODO parameterize id allocation?
+                                  (let [id-prefix (str (.getYear time))]
+                                    (format-external-id (allocate-external-id! id-prefix))))
```

Comment:
I would like rows 939 and 940 to be in format-external-id function (rename maybe the fn?). Unnecessary details here what the external id consists of imho.